cql/src/main/java/io/stargate/cql/CqlActivator.java

```diff
@@ -99,6 +99,19 @@ private static Config makeConfig() {
       c.rpc_address = listenAddress;
       c.native_transport_port = cqlPort;
 
+      c.native_transport_max_concurrent_connections =
+          Long.getLong("stargate.cql.native_transport_max_concurrent_connections", -1);
+      c.native_transport_max_concurrent_connections_per_ip =
+          Long.getLong("stargate.cql.native_transport_max_concurrent_connections_per_ip", -1);
+      c.native_transport_max_concurrent_requests_in_bytes =
+          Long.getLong(
+              "stargate.cql.native_transport_max_concurrent_requests_in_bytes",
+              Runtime.getRuntime().maxMemory() / 10);
```

Comment:
This is what OSS Cassandra uses ([source](https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/config/DatabaseDescriptor.java#L534-L542)). We might want to lower them in Stargate since there are potentially other workloads running in the same VM.

Reply:
Good point, but the more I think about how much to lower it the more I think we should go with the Cassandra defaults.. Although 10% seems a lot, this is just a max and it's not like occupy this amount all the time.. Not sure, if you decide to decrease, then I would say max by factor 2..